src/components/__tests__/Bullet.ts

```diff
@@ -143,12 +141,26 @@ describe.skip('expansion', () => {
       setCursor(['a', 'b']),
     ])
 
-    const subthoughts = await findSubthoughts('a')
-    const bulletsOfSubthoughtsA = await findAllByLabelText(subthoughts[0], 'bullet')
-    userEvent.click(bulletsOfSubthoughtsA[0])
+    const initialState = exportContext(store.getState(), [HOME_TOKEN], 'text/plain')
 
-    const thoughtCursor = await findCursor()
-    expect(thoughtCursor).toHaveTextContent('a')
+    // Find all bullet elements:
+    const bulletElements = await screen.findAllByLabelText('bullet')
+
+    // Click the second bullet directly:
+    await userEvent.click(bulletElements[1]) // Index 1 for the second bullet
```

Comment:
Selecting all bullets and then indexing into the array is not ideal, because it couples the test to the linear order of thoughts as they are rendered from top-to-bottom on the screen. If a thought is the 2nd or 4th or 10th from the top, that's not very meaningful to the user. While it's not much of a problem when only rendering a few thoughts, it's going to become a problem for tests that involve a larger number of thoughts, or deeply nested thoughts. Pretty soon you have developers counting lines of code in order to figure out which `thoughtContainers[12]` refers to. And adding one thought at the top changes the indexes all the way down.

Instead, we want to find a way to select a thought based on its text. You can use [contextToPath(['a', 'b'])](https://github.com/cybersemics/em/blob/d90f6562fc3cab351aa6aa70adaf1060d51bad4c/src/selectors/contextToPath.ts) to get the path which can be used to select the exact testid of interest. Or you could select by the visible text itself, or embed the thought value in a data attribute. They each have their tradeoffs, but at least they are not dependent on the linear order.

Let me know what you think. Thanks!